snuba/clickhouse/astquery.py

```diff
@@ -52,6 +51,9 @@ def __init__(self, query: Query, settings: RequestSettings,) -> None:
         self.__settings = settings
         self.__formatted_query: Optional[str] = None
 
+    def get_applied_sampling_rate(self) -> Optional[float]:
+        return self.__sample
```

Comment:
Is "applied" necessary in this name? It seems like at this level, the sampling rate either exists or it doesn't and how it's set doesn't seem particularly relevant. (I guess this comment really applies to the abstract base class and it's descendants but I'm too lazy to move it.)